private/numerics/functions/elliptic.rkt

```diff
@@ -0,0 +1,237 @@
+#lang racket/base
+
+;; Consult "Numerical Computation of Real or Complex Elliptic
+;; Integrals" Carlson (1994) http://arxiv.org/pdf/math/9409227v1.pdf
+;;
+;; The algorithm in scmutils seems similar to what is in that paper,
+;; but slightly different.
+;;
+;; Consider also exporting under the more familiar names Rf, Rj, Rc,
+;; Rd. Is there any input method that gets those names as subscripts.
+
+(require
+ (only-in mechanics π π/2 machine-ε))
+
+(require
+ (only-in racket/math cosh tanh))
+
+(require
+ (only-in racket/fixnum fx< fx+))
+
+(define (Rf x y z)
+  (define ε (expt (machine-ε) (/ 1 6)))
```

Comment:
Can you get the subscript f to work in emacs? I'd prefer subscript F, but it [looks like its not available in unicode](http://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts#Other_superscript_and_subscript_characters). Subscript f should its not for my Emacs version 24.4.